server-common/src/main/java/org/apache/kafka/purgatory/DeferredEventPurgatory.java

```diff
@@ -26,10 +26,9 @@
 
 /**
  * The purgatory which holds events that have been started, but not yet completed.
```

Comment:
Minor but can we avoid the use of the word purgatory? It is not accurate for what this type and package are doing.

Reply:
Why do you think that it is not accurate? For me, a purgatory is a data structure holding something which is completed later. That seems to fit here.

Reply:
Some of my thoughts: the founder of Kafka had some artistic elements, with Kafka(a novelist) mainly representing the modernist school, and "purgatory" being a term used in Dante's "Divine Comedy", a representative writer of the Renaissance era. Purgatory' is first used in the `TimingWheel` algorithm and has been used several times to represent  a `ThresholdFuture`  data structure.